ol_stock_constrained_availability/models/stock_move.py

```diff
@@ -0,0 +1,38 @@
+# Import Odoo Libs
```

Comment:
@Vandan-OSI I did some code formatting and during that I noticed that this class isn't in the init py, should it be or should this file be deleted?